src/components/bookList.js

```diff
@@ -29,27 +29,33 @@ const BookList = () => {
     <div className="imp-book">
       {book.map((key) => (
         <div key={key.item_id} className="book-cont">
-          <div>
-            <h4>{key.title}</h4>
-            <p>{key.author}</p>
-            <button type="button">comments</button>
-            <button type="button" onClick={() => handleremoveBook(key.item_id)}>remove</button>
-            <button type="button">edit</button>
+          <div className="author-section">
+            <div className="book-author">
+              <p className="cat">{key.category}</p>
+              <h4 className="aut">{key.title}</h4>
+              <p className="tit">{key.author}</p>
+            </div>
+            <div className="book-btn">
+              <button type="button" className="l-btn">comments</button>
+              <button className="l-btn rem" type="button" onClick={() => handleremoveBook(key.item_id)}>remove</button>
+              <button type="button" className="l-btn">edit</button>
+            </div>
           </div>
-          <div>
-            <AiOutlineLoading3Quarters />
-            <div>
-              <span>64%</span>
-              <span>completed</span>
+          <div className="comp-icon">
+            <div className="progress">
+              <AiOutlineLoading3Quarters />
+            </div>
+            <div className="prog-details">
+              <span className="comp-percent">64%</span>
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up: there is a border separator between the percentage and the current chapter section. Kindly add that :+1:  check below :point_down:  the comparison between your design and [Zaplin](https://zeplin.io/) design :slightly_smiling_face: 

| Your Design | Zaplin Design|
|--------|--------|
| ![Screenshot from 2023-05-28 03-58-21](https://github.com/emmanuelsteven/BookStore-React-Redux/assets/45226848/f2284a65-934d-4bc9-bfb8-5f3b1a16f388) | ![Screenshot from 2023-05-28 03-59-21](https://github.com/emmanuelsteven/BookStore-React-Redux/assets/45226848/3a889dfe-cf1d-488e-9a86-dc919564e885) |